spec/pg_easy_replicate/orchestrate_spec.rb

```diff
@@ -190,18 +204,9 @@
   describe ".drop_subscription" do
     before do
       PgEasyReplicate.bootstrap({ group_name: "cluster1" })
-
-      described_class.create_publication(
-        group_name: "cluster1",
-        conn_string: connection_url,
-      )
     end
```

Comment:
Removed this as we are testing the drop_subscription in isolation here so dont think it's needed.